R/nested_resampling.R

```diff
@@ -0,0 +1,191 @@
+#' @title Extract Inner Tuning Results
+#'
+#' @description
+#' Extract inner tuning results of nested resampling. Implemented for
+#' [mlr3::ResampleResult] and [mlr3::BenchmarkResult]. The function iterates
+#' over the [AutoTuner] objects and binds the tuning results to a
+#' [data.table::data.table()]. [AutoTuner] must be initialized with
+#' `store_tuning_instance = TRUE` and `resample()` or `benchmark()` must be
+#' called with `store_models = TRUE`.
+#'
+#' @section Data structure:
+#'
+#' The returned data table has the following columns:
+#'
+#' * `experiment` (integer(1))\cr
+#'   Experiment of the benchmark grid.
```

Comment:
```suggestion
#'   Index, giving the according row number in the original benchmark grid.
```
Maybe something like this is a bit more clear.